src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                      "geologic feature"@en ;
+                       skos:scopeNote "Geologic Features include sedimentary basins, stratigraphic units, non-stratigraphic (lithodemic) units, stratigraphic event features, provinces, tectonic and structural features, georesource accumulations, and geologically significant sites, among others"@en .
 
 
 ###  http://sweetontology.net/realmGeol/GeologicProvince
 soreag:GeologicProvince rdf:type owl:Class ;
-                      rdfs:subClassOf soreag:GeologicFeature ;
-                      rdfs:label "geologic province"@en .
+                        rdfs:subClassOf soreag:TectonicEntity ;
+                        dcterms:contributor <http://linked.data.gov.au/def/gsq> ;
+                        skos:definition "Geologic provinces are tectonic entities that include cratons/shields (in the continental realm), orogens, sedimentary basins, and tectonised/metamorphosed and/or mineralised regions, as well as large igneous provinces (LIPs). (Definition created here)"@en ;
+                        skos:prefLabel "geologic province"@en .
 
 
 ###  http://sweetontology.net/realmGeol/GeologicStructure
 soreag:GeologicStructure rdf:type owl:Class ;
-                       rdfs:subClassOf sorea:Geosphere ,
-                                       [ rdf:type owl:Restriction ;
-                                         owl:onProperty sorel:hasRealm ;
-                                         owl:allValuesFrom sorea:Geosphere
-                                       ] ;
-                       rdfs:label "geologic structure"@en .
+                         rdfs:subClassOf sorea:Geosphere ,
+                                         [ rdf:type owl:Restriction ;
+                                           owl:onProperty sorel:hasRealm ;
+                                           owl:allValuesFrom sorea:Geosphere
+                                         ] ;
+                         rdfs:label "geologic structure"@en .
+
+
+###  http://sweetontology.net/realmGeol/GeologicalHeritageSite
+soreag:GeologicalHeritageSite rdf:type owl:Class ;
+                              rdfs:subClassOf soreag:GeologicallySignificantSite ;
+                              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                              dcterms:source [ ] ,
+[ ] ;
+skos:definition "Geological Heritage Sites (formerly known as Geological Monuments) display geological features that are so outstanding or so rare that they form part of our natural heritage, requiring recognition, description and protection, for education, research and their geotourism potential. They show evidence of the geological processes that have occurred on Earth, and of the plants and animals that have lived on it. These sites include landforms, rock outcrops, river banks, sea cliffs and shore platforms, and are, in part, represented in road cuttings, mines, quarries and other excavations. (Geological Society of Australia)"@en ;
```

Comment:
Maybe SWEET should have a class for 'Site' as a generic human-defined location.
Is a site a GeologicFeature?  I think not-- a soreag:GeologicallySignificantSite exists because of a human assertion; the location does include outcrop of geologicFeature(s), but its identification as an entity is not inherent in any geologic feature.  If it weren't marked as a GeologicallySignificantSite, a geologists mapping in the area could recognize the features exposed there, but not that it is a 'GeologicallySignificantSite'